src/components/Post.js

```diff
@@ -2,24 +2,33 @@ import { useState } from "react";
 import axiosInstance from "../api/axios";
 import img1 from "../images/img1.jpg";
 import { useNavigate } from "react-router-dom";
+// import axios from "axios";
 
 const Post = () => {
   const [blog, setBlog] = useState({ title: "", content: "", author: "" });
   const [postErr, setPostErr] = useState("")
+  // const [isAuth, setIsAuth] = useState("") //state to check if user have access to post
 
 
+  
   const handleBlogDetails = (e) => {
     e.preventDefault();
-    setBlog((current) => ({ ...current, [e.target.name]: e.target.value }));
+    setBlog((prev) => ({ ...prev, [e.target.name]: e.target.value }));
+    // setIsAuth("You are not allowed to post until you are logged in")
   };
 
   let navigate = useNavigate();
   const handleSubmit = async (e) => {
     e.preventDefault();
     try {
-      const result = await axiosInstance.get("/", blog, {
-
-      });
+      const result = await axiosInstance.get(blog, {
+        method: "GET",
```

Comment:
Why set `method` again, when you have `axiosInstance.get`?

Reply:
I know this is not right, am still debugging this part because the access token am trying to get is not giving me the desired result so trying all possible ways to see if it will yield. am getting a 401 error with "Authorization header must contain two space-delimited values"

Reply:
Have you spoken to the backender?

Reply:
Yes they are working on it